plugins/labhub.py

```diff
@@ -387,3 +388,65 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'pr\s+list\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)',  # Ignore LineLengthBear, PyCodeStyleBear
```

Comment:
i think we can just use repo name. We are discarding the `host` and `org` either way. And this way we'd only need to give repo name and not the complete url. 

Just like `corobo new issue <repo> ..`